src/main/java/cn/nukkit/block/BlockDirt.java

```diff
@@ -54,6 +56,9 @@ public String getName() {
     @Override
     public boolean onActivate(Item item, Player player) {
         if (item.isHoe()) {
+            if(player != null){
+                player.getLevel().addSound(player, Sound.USE_GRASS);
```

Comment:
Probably better to play the sound AFTER the action is executed for consistent responses